scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -1182,8 +1182,21 @@ def cmake_assemble_args(extra_args, handler, extra_conf_files, extra_overlay_con
         return args_expanded
 
     def cmake(self, filter_stages=[]):
+        args = []
```

Comment:
We should add some documentation on this new filter feature for twister, it seems like a useful addition

Reply:
yes, I need add document